### ISSUE TITLE
dataset unreachable

### ISSUE THREAD

**Issue by mtxing69** [datitran/face2face-demo#4]
The links to video and train dataset are not reachable, anyone knows why?

**Comment by jasonmayes** [comment]
+1 all links to drop box files, video, dataset, etc are 404.

Can you re upload please? Thanks!

**Comment by julien2512** [comment]
I've found this : http://dlib.net/files/data/ibug_300W_large_face_landmark_dataset.tar.gz
from https://github.com/davisking/dlib/issues/359

**Comment by julien2512** [comment]
There it is
http://dlib.net/files/shape_predictor_68_face_landmarks.dat.bz2

**Comment by DeepInEvil** [comment]
The pre-trained model is not available, kindly check

**Comment by julien2512** [comment]
I had to change from menpo::tbb to conda-forge:tbb in environment.yml.
Then I used skvideo.io rather than cv2 in generate_train_data.py.
But it was a too old release, and I need to add make it compatible for utf-8 and bytes, adjust the indentation level (in a very very gross fashion), use the 2nd stream from the videos.
I also add many codecs to support my video (maybe it is not usefull).

The result of my first try is here : https://media.giphy.com/media/3ohs7QDestHIudFMoE/giphy.gif
I used this video from youtube : https://www.youtube.com/watch?v=PXQh9jTwwoA on 480 x 360.

I mean, you don't really need the original video. The shape predictor is enough.
